include/plotIt.h

```diff
@@ -57,6 +57,19 @@ namespace plotIt {
       return errors_type;
   }
 
+  enum Log {
```

Comment:
I wanted to use implicit int conversion, see https://github.com/cp3-llbb/plotIt/pull/15/files#diff-8ea05427ed1e264915a5a24ee5a597b5R425